src/main/java/frc/robot/Constants.java

```diff
@@ -71,6 +71,22 @@ public final class Constants {
       public static final float ARM_MAX_POSITION = 0;
       public static final float ARM_MIN_POSITION = 0;
 
+      //floor arm constants
+      public static final int FLOOR_ARM_SOLENOID_CHANNEL = 0;
+      public static final double FLOOR_ARM_POWER_SCALING = 0;
+      public static final int FLOOR_ARM_MOTOR = 13;
```

Comment:
Needs a unique CAN id

Reply:
Set the CAN id to 14 and changed the ID's for the FloorIntake and the ArmIntake to be 15 and 16 respectively.